neo/Consensus/ConsensusContext.cs

```diff
@@ -234,9 +208,6 @@ public ConsensusPayload MakePrepareRequest()
 
         public ConsensusPayload MakeRecoveryMessage()
         {
-            var changeViewPayloads = TransactionHashes == null || PreparationPayloads.Count(p => p != null) < M
```

Comment:
It was more efficient to have this optimization. Are you taking it away because the code is simpler ifi t always includes the `ChangeView` payloads if they are present?

Reply:
I guess it is fine to always replay them, because it guarantees it runs the exact same messages during recovery that it will run in normal operation. Maybe slightly less efficient, but it has the guarantee of the exact same behavior in recovery as it would have during normal receiving of the messages.

Reply:
I always replay them because it would be easier to continue changing the view after receiving the `RecoveryMessage`.

However, there is still a place to optimize, that is, if the `ViewNumber` of the `RecoveryMessage` to be constructed is consistent with the requested view number, it is not necessary to include the `ChangeView` payloads.